swarm/api/inspector.go

```diff
@@ -0,0 +1,58 @@
+// Copyright 2016 The go-ethereum Authors
```

Comment:
2019

Reply:
i really dont like inspector. please use standard names like 
debug, monitoring,

Reply:
I agree with @nonsense, although while I like `Inspector` and I do think it is a good one, I am not attached to it - as long as it is not `Debug`, `Monitoring` or `Management`.